src/pages/_layout.svelte

```diff
@@ -19,19 +19,21 @@
     return $me.id
   }
 </script>
-
-<Bootstrap />
-
-<Nav user={$me} />
-
-<main class="container-xl">
-  <Error />
-
-  {#if $loggingOut}
-     <h4>Thank you for visiting, we are signing you out now...</h4>
-  {/if}
-
-  <slot />
-</main>
-
-<Footer />
+<!-- This div prevents material from messing up the layout. Once converted to material grid it can be removed -->
+<div class='w-100'>
+  <Bootstrap />
+  
+  <Nav user={$me} />
+  
+  <main class="container-xl">
+    <Error />
+    
+    {#if $loggingOut}
+    <h4>Thank you for visiting, we are signing you out now...</h4>
```

Comment:
VSCode's poor handling of indention must have done this.
```suggestion
      <h4>Thank you for visiting, we are signing you out now...</h4>
```